pkg/apis/monitoring/v1alpha1/alertmanager_config_types.go

```diff
@@ -1132,6 +1136,137 @@ type MSTeamsConfig struct {
+}
+
+// Validate ensures JiraField is valid.
+func (jf *JiraField) Validate() error {
+	if jf.Value.Raw == nil {
+		return nil
+	}
+
+	var ret interface{}
+	dec := json.NewDecoder(bytes.NewBuffer(jf.Value.Raw))
+	dec.DisallowUnknownFields()
+	if err := dec.Decode(&ret); err != nil {
+		return fmt.Errorf("Value json decode: %w", err)
+	}
+	return nil
+}
+
+// JiraConfig configures notifications via Jira.
+// See https://prometheus.io/docs/alerting/latest/configuration/#Jira_config
+// It requires Alertmanager >= 0.28.0.
+type JiraConfig struct {
+	// sendResolved defines whether to notify about resolved alerts.
+	//
+	// +optional
+	SendResolved *bool `json:"sendResolved,omitempty"`
+
+	// apiURL defines the Jira API URL i.e. https://company.atlassian.net/rest/api/2/
+	// The full API path must be included.
+	// If not specified, default API URL will be used.
+	//
+	// +kubebuilder:validation:Pattern:="^http(s)?://.+$"
+	// +optional
+	APIURL *string `json:"apiURL,omitempty"`
+
+	// project defines the project key where issues are created.
+	//
+	// +kubebuilder:validation:MinLength=1
+	// +required
+	Project string `json:"project"`
+
+	// summary defines the issue summary template.
+	//
+	// +optional
+	Summary *string `json:"summary,omitempty"`
+
+	// description defines the issue description template.
+	//
+	// +optional
+	Description *string `json:"description,omitempty"`
+
+	// labels defines labels to be added to the issue.
+	//
+	// +kubebuilder:validation:MinItems=1
+	// +kubebuilder:validation:items:MinLength=1
+	// +listType=set
+	// +optional
+	Labels []string `json:"labels,omitempty"`
+
+	// priority defines the priority of the issue.
+	//
+	// +optional
+	Priority *string `json:"priority,omitempty"`
+
+	// issueType defines a type of the issue (e.g. Bug).
+	//
+	// +kubebuilder:validation:MinLength=1
+	// +required
+	IssueType string `json:"issueType"`
+
+	// resolveTransition defines name of the workflow transition to resolve an issue.
+	// The target status must have the category "done".
+	// NOTE: The name of the transition can be localized and depends on the language setting of the service account.
+	//
+	// +optional
+	ResolveTransition *string `json:"resolveTransition,omitempty"`
+
+	// reopenTransition defines name of the workflow transition to reopen an issue.
+	// The target status should not have the category "done".
+	// NOTE: The name of the transition can be localized and depends on the language setting of the service account.
+	//
+	// +optional
+	ReopenTransition *string `json:"reopenTransition,omitempty"`
+
+	// wontFixResolution defines if reopenTransition is defined, ignore issues with that resolution.
+	//
+	// +optional
+	WontFixResolution *string `json:"wontFixResolution,omitempty"`
+
+	// reopenDuration defines to reopen the issue when it is not older than this value (rounded down to the nearest minute).
+	// The 'resolutiondate' field in Jira is used to determine the age of the issue.
```

Comment:
```suggestion
	// The 'resolution date' field in Jira is used to determine the age of the issue.
```
Hi seems there new spell check workflow got this one.

Reply:
Done, please have a look.